qemu/tests/balloon_check.py

```diff
@@ -383,19 +383,19 @@ def error_report(self, step, expect_value, monitor_value, guest_value):
         :param guest_value: memory size report from guest, this value can be
                             None
         """
-        pass
+        raise NotImplementedError
 
     def get_memory_status(self):
         """
         Get Memory status inside guest.
         """
-        pass
+        raise NotImplementedError
 
     def _balloon_post_action(self):
         """
         Wait for guest memory goes into stable status
         """
-        pass
+        raise NotImplementedError
```

Comment:
Looks have to keep the original implementation for this method to let the test pass.